autokeras/tasks/time_series_forecaster.py

```diff
@@ -122,7 +124,7 @@ class TimeseriesForecaster(SupervisedTimeseriesDataPipeline):
         predict_until: Int. The end point of the forecast for each sample (in number
             of steps) after the last time step in the input. If N is the last step in
             the input, then the last step of the predicted output will be
-            N + predict_until. Defaults to 10.
+            N + predict_until. Defaults to None.
```

Comment:
Please describe how long will it predict if unspecified with
"If unspecified, it will ..."